### PR TITLE
Added support for playing srcObject in video elements (the dirty way,…

### PR DESCRIPTION
**Pull request by stefhak**
… relying on CreateObjectURL).

### CODE REVIEW

File: bridge/client/webrtc.js [hunk 1]
```diff
@@ -1656,6 +1656,18 @@
...
+            console.log("getting video element's srcObject");
+            return this._srcObject;
+        },
+        "set":  function (stream) {
+            console.log("setting video element's srcObject to: " + stream);
+            this._srcObject = stream;
+            this.src = url.createObjectURL(stream);
+        }
+    }); 
+
```
Comment (adam-be) [inline bridge/client/webrtc.js]: This looks good. Please remove the comments and the double white space after "set": then we're good to go.

**Comment by stefhak** [comment]
Made the proposed changes. Possible to merge, or should I make a single commit out of the change?

**Comment by superdump** [comment]
If it were me, I would squash or fix the second commit into the first to have one clean commit that is correct from the beginning. I generally iterate through reviews that way - isolated commits that implement one small logical grouping of work and then write fixes against each of those commits and squash / fix them into those commits.

**Comment by stefhak** [comment]
@superdump if it were you, would you have done something like I did here?

**Comment by superdump** [comment]
Almost, I wouldn't have noted the correction. It's fine that the commit looks like it was perfect from the start. :smile: The reason is that it makes the git history cleaner, and the discussion of the review process is documented here.

**Comment by stefhak** [comment]
@superdump is "almost" close enough :) or is there something else to be done before someone could consider merging?

**Comment by superdump** [comment]
I'll merge when I get to the office. Thanks! :smile:

**Pull request closed without merging**